keyboards/monarch/keymaps/via/config.h

```diff
@@ -0,0 +1 @@
+#define DYNAMIC_KEYMAP_EEPROM_MAX_ADDR 2047
```

Comment:
```suggestion
#pragma once

#define DYNAMIC_KEYMAP_EEPROM_MAX_ADDR 2047
```

Also needs licence header

Reply:
Made updates